src/OpenTelemetry.Instrumentation.AdditionalDeps/Package.xml

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RestoreSources>https://api.nuget.org/v3/index.json;$(TracerHomePath)\AdditionalDeps;</RestoreSources>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Condition="'$(TracerHomePath)' != ''" Include="OpenTelemetry.Instrumentation.AdditionalDeps" Version="*" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
I *think* I understand the effect of this when building a NuGet package but I'm not sure of what it does when building a store. Could you clarify that?

Reply:
Good point, I added it for test and missed removing it. This is not needed.